test-it/testList.json

```diff
@@ -331,14 +331,18 @@
                 "SubVIParameters.via",
                 "SubVISimple.via",
                 "TemplateBasics.via",
+                "TestVI_ArrayDataTypeCoerecedToVariant.via",
                 "TestVI_ArrayDelete.via",
                 "TestVI_ArrayDelete_1D_DifferentIndices.via",
                 "TestVI_ArrayDelete_2D_DifferentIndices.via",
                 "TestVI_ArrayInsertPartial.via",
                 "TestVI_ArrayNonFlatInflate.via",
                 "TestVI_ArrayOfRefs.via",
                 "TestVIArrayMaxAndMin.via",
+                "TestVI_ArraysWiredToVariantToData",
```

Comment:
Missing the via extension

Reply:
Thanks for catching this, and pushing a fix!